app/models/user.rb

```diff
@@ -5,7 +5,11 @@ class User < ApplicationRecord
   validates :name, presence: true
   validates :posts_counter, numericality: { greater_than_or_equal_to: 0 }
 
+  # def three_recent_posts
```

Comment:
Please erase commented code from here and other files as it will make the overall project.